source/geometry_model/box.cc

```diff
@@ -21,6 +21,8 @@
 
 #include <aspect/geometry_model/box.h>
 #include <aspect/geometry_model/initial_topography_model/zero_topography.h>
+#include <aspect/geometry_model/initial_topography_model/interface.h>
```

Comment:
I do not think you need this include, because you include `zero_topography` anyway.